### PR TITLE
Adding docstrings to Keras Core ops in numpy.py file

### PR DESCRIPTION
**Pull request by sukhdeepg**

### CODE REVIEW

File: keras_core/ops/numpy.py [hunk 1]
```diff
@@ -146,8 +146,20 @@
 
 
 def broadcast_shapes(shape1, shape2):
-    # Broadcast input shapes to a unified shape.
-    # Convert to list for mutability.
+    """Broadcast input shapes to a unified shape.
+    Convert to list for mutability.
```
Comment (fchollet) [inline keras_core/ops/numpy.py:150]: Add line break above

Comment (sukhdeepg) [reply keras_core/ops/numpy.py:151]: done, I've added blank line above "convert to list ..."

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Comment by sukhdeepg** [comment]
could you please give approval for approval? I'm getting the above message
Screenshot 2023-07-17 at 8 04 50 AM

I just merged code from main into my branch.

**Pull request merged**